moduleframework/helpers/openshift_helper.py

```diff
@@ -54,8 +58,83 @@ def __init__(self):
         # application name is taken from docker.io/modularitycontainer/memcached
         self.app_name = self.container_name.split('/')[-1]
         self.app_ip = None
+        random_str = ''.join(random.choice(string.lowercase) for _ in range(3))
+        self.project_name = "{project}-{random_str}".format(project=self.app_name,
+                                                            random_str=random_str)
         common.print_debug(self.icontainer, self.app_name)
 
+    def _get_openshift_ip_registry(self):
+        """
+        Function returns an IP of OpenShift registry.
+        Command which is used for getting this info is `oc get svc -n default docker-registry`
+        :return: str or None
+        """
+        openshift_ip_register = None
+        docker_registry = self.runHost('oc get svc -n default %s -o json' % common.OPENSHIFT_DOCKER_REGISTRY,
+                                       ignore_status=True).stdout
+        try:
+            docker_registry = json.loads(docker_registry)
+            openshift_ip_register = docker_registry.get("spec").get("clusterIP")
+        except AttributeError:
+            pass
+        return openshift_ip_register
+
+    def _change_openshift_account(self, account="system:admin", password=None):
+        """
+        Function switches to specific account inside OpenShift environment
+        :param account: Either user specified account or 'system:admin'
+        :param password: Either user specified account
+        """
+        if password is None:
+            s = self.runHost("oc login -u %s" % account, verbose=common.is_not_silent())
+        else:
+            s = self.runHost("oc login -u %s -p %s" % (account, password), verbose=common.is_not_silent())
+
+
+    def _register_docker_to_openshift_register(self):
+        """
+        Function registers an OpenShift docker-registry into docker
```

Comment:
it actually just pushes the container image inside OpenShift's registry; you don't *register registry into docker*